index.d.ts

```diff
@@ -570,9 +570,9 @@ declare module 'eris' {
   }
 
   export class Collection<T extends { id: string }> extends Map<string, T> {
-    baseObject: new () => T;
+    baseObject: new (...args: any[]) => T;
     limit?: number;
-    constructor(baseObject: new () => T, limit?: number);
+    constructor(baseObject: new (...args: any[]) => T, limit?: number);
```

Comment:
The typings use `Array<any>` in most places, you should keep this in mind

Reply:
using `Array` for simple types is meme tho

Reply:
Then make a PR changing all array types I used to `type[]` 🙃

Reply:
me too thanks

Reply:
type[] is standard for simple types in tslint

Reply:
@aequasi I'm fully aware of that, couldn't break my habit of using Array<T> before. I might PR the change at some point, unless someone else wants to do it.